forms.py

```diff
@@ -30,15 +30,25 @@ class ReplyForm(FlaskForm):
     submit = SubmitField("Send message")
 
 class EmployeeForm(FlaskForm):
-    first_name = StringField("First name: ", default="", validators=[InputRequired()])
-    last_name = StringField("Last name: ", default="", validators=[InputRequired()])
-    email = StringField("Email Address", [validators.Length(min=6, max=100)])
+    first_name = StringField("First name ", validators=[InputRequired()])
+    last_name = StringField("Last name", validators=[InputRequired()])
+    email = StringField("Email Address",[validators.Length(min=6, max=100)])
+    address = StringField("Address", default="", validators=[InputRequired()])
+    bio = TextAreaField("About Info", widget=TextArea())
     role = StringField("Role: ", default="", validators=[InputRequired()])
     access_level = SelectField("Choose an option", 
                                         choices = [("managerial", "Managerial"),
                                                     ("ordinary staff", "Ordinary staff")], validators=[InputRequired()])
     submit = SubmitField("Create")
 
+class ProfileForm(FlaskForm):
+    email = StringField("Email Address")
```

Comment:
This is used to edit a staff memebers profile, right? What's the primary key in this table if its not the email - which has no inputRequired if it is the primary key?

Reply:
> This is used to edit a staff memebers profile, right? What's the primary key in this table if its not the email - which has no inputRequired if it is the primary key?

UGH STOP THIS IS WHY I NEED PEOPLE TO WATCH WHAT I'M DOING. brb gonna add that validators=[InputRequired()] rn !!!

(and yes it's used to edit a staff member's profile)